lib/stylesheet/manager.rb

```diff
@@ -196,10 +196,11 @@ def stylesheet_preload_tag(target = :desktop, media = 'all')
     end.join("\n").html_safe
   end
 
-  def stylesheet_link_tag(target = :desktop, media = 'all')
+  def stylesheet_link_tag(target = :desktop, media = 'all', preload_callback = nil)
     stylesheets = stylesheet_details(target, media)
     stylesheets.map do |stylesheet|
       href = stylesheet[:new_href]
+      preload_callback.call(href, 'style') if preload_callback
```

Comment:
This needs a test as well, current `stylesheet_link_tag` won't cover the preload callbacks.